### PR TITLE
Daniel Lounsbury

### PR DESCRIPTION
**Pull request by BigSkyCoder2003**
Add my card information

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -113,6 +113,67 @@ <h2>A project for first-time contributions</h2>
...
+                <a href="https://www.cpuid.com/softwares.html" target="_blank" title="Second Resource">Local Hardware Monitoring and Stress Testing Software</a>
+              </li>
+              <li>
+                <a href="https://www.youtube.com/watch?v=dQw4w9WgXcQ" target="_blank" title="Third resource">Good Talk</a>
+              </li>
+            </ul>
+          </div>
+        </div>
+        <!-- ________ *TEMPLATE: MAKE A COPY* Contributor card END ________  -->
+
```
Comment (malakada) [inline index.html:142]: Looks like you accidentally duplicated another template block here which would be very confusing for future contributors (like yourself!) who want to put their card in the right place.

Can you please remove this extra template so we can get your PR merged? :D

**Review by malakada — CHANGES_REQUESTED** [review]
Awesome job @BigSkyCoder2003! Your card looks great, and I'd love to get your merged in... but there's extra templating code that looks like you accidentally duplicated into the file. 😅 

These things happen all the time, it's part of why we have a code review process. 😄 Just delete the extra empty template, make a new commit, and push it up. Easy peasy. 💪 

We'll come by and look at it again to see if you've been able to update it, but if you're feeling excited feel free to ping in the Discord.

**Comment by BigSkyCoder2003** [comment]
Should have that fixed now! Please let me know if there is anything else I should do! Thank you for your feedback!

File: index.html [hunk 2]
```diff
@@ -149,6 +149,33 @@ <h2>A project for first-time contributions</h2>
...
+        <!-- ________ Daniel Lounsbury CARD START ________  -->
+        <div class="card">
+          <p class="name">Daniel Lounsbury</p>
+          <p class="contact">
+            <i class="fa-brands fa-instagram"></i>
+            <a href="https://www.instagram.com/" target="_blank">daniel.loun</a>
+            <i class="fab fa-github"></i>
+            <a href="https://www.github.com" target="_blank">BigSkyCoder2003</a>
+          </p>
+          <p class="about">I am a person</p>
```
Comment (malakada) [inline index.html:161]: lmao, love it haha

File: index.html [hunk 3]
```diff
@@ -149,6 +149,33 @@ <h2>A project for first-time contributions</h2>
 
         <!-- ========= Paste YOUR CARD directly BELOW this line ========= -->
 
+        <!-- ________ Daniel Lounsbury CARD START ________  -->
+        <div class="card">
+          <p class="name">Daniel Lounsbury</p>
+          <p class="contact">
+            <i class="fa-brands fa-instagram"></i>
+            <a href="https://www.instagram.com/" target="_blank">daniel.loun</a>
```
Comment (malakada) [inline index.html:157]: Hey! This link doesn't go to your profile! Can you update it? :S

File: index.html [hunk 4]
```diff
@@ -149,6 +149,33 @@ <h2>A project for first-time contributions</h2>
...
+            <i class="fab fa-github"></i>
+            <a href="https://www.github.com" target="_blank">BigSkyCoder2003</a>
```
Comment (malakada) [inline index.html:159]: Same here -- link doesn't go to your profile.

File: index.html [hunk 5]
```diff
@@ -149,6 +149,33 @@ <h2>A project for first-time contributions</h2>
...
+            <p>3 Useful Dev Resources</p>
+            <ul>
+              <li>
+                <a href="https://benddevops.org/" target="_blank" title="First Resource">Bend DevOps</a>
+              </li>
+              <li>
+                <a href="https://www.cpuid.com/softwares.html" target="_blank" title="Second Resource">Local Hardware Monitoring and Stress Testing Software</a>
+              </li>
+              <li>
+                <a href="https://www.youtube.com/watch?v=dQw4w9WgXcQ" target="_blank" title="Third resource">Good Talk</a>
```
Comment (malakada) [inline index.html:172]: 🏆

**Review by malakada — CHANGES_REQUESTED** [review]
Sorry I didn't notice it the first time around, but you should update your links to point to your profile. :)

**Review by TenTraicion — APPROVED** [review]

**Comment by TenTraicion** [comment]
@BigSkyCoder2003, Congratulations on your PR Merge, and Thanks for your Contribution. You can check your card on our [Live Website](https://syknapse.github.io/Contribute-To-This-Project)! It may take 2-3 minutes to make your card visible on our website. 

Welcome to the Open Source Community. You can follow the [Next Steps](https://github.com/Syknapse/Contribute-To-This-Project#next-steps) now. You are also welcome to join our [Discord Server: https://discord.gg/tWkvS4ueVF](https://discord.gg/tWkvS4ueVF) and be a part of our community.

**Pull request merged**